saltproc/whole_core_axial_ref

```diff
@@ -0,0 +1,726 @@
+% --- Surfaces (core pitch = 10.16 cm): Zone I-A
```

Comment:
does this need to be in the top level directory? Is there a better place to keep these?